_includes/specials/taproot/en/21-thanks.md

```diff
@@ -0,0 +1,509 @@
+support in libsecp256k1 (used by Bitcoin Core) or alternative node software.
+
+{:#taproot-activation-discussion}
+**Taproot activation discussion**
+
+As the taproot implementation was merged into Bitcoin Core, it fell on
+the community to decide how it would be activated.  This led to several
+months of discussion, with the most active conversations on the taproot
+activation IRC channel between the following users, developers, and
+miners:
+
+<i>
+_6102bitcoin,
+AaronvanW,
+achow101,
+aj,
+alec,
+Alexandre_Chery,
+Alistair_Mann,
+amiti,
+_andrewtoth,
+andrewtoth,
+andytoshi,
+AnthonyRonning,
+ariel25,
+arturogoosnargh,
+AsILayHodling,
+averagepleb,
+bcman,
+belcher,
+benthecarman,
+benthecarman81,
+Billy,
+bitcoinaire,
+bitentrepreneur,
+bitsharp,
+bjarnem,
+blk014,
+BlueMatt,
+bobazY,
+brg444,
+btcactivator,
+btcbb,
+cato,
+catwith1hat,
+cguida,
+CodeShark___,
+conman,
+copumpkin,
+Crash78,
+criley,
+CriptoLuis,
+CubicEarth,
+darbsllim,
+darosior,
+Day,
+DeanGuss,
+DeanWeen,
+debit,
+Decentralizedb,
+devrandom,
+DigDug,
+dome,
+dr_orlovsky,
+duringo,
+dustinwinski,
+eeb77f71f26eee,
+eidnrf,
+elector,
+elichai2,
+Emcy,
+emzy,
+entropy5000,
+eoin,
+epson121,
+erijon,
+eris,
+evankaloudis,
+faketoshi,
+fanquake,
+fedorafan,
+fedorafansuper,
+felixweis,
+fiach_dubh,
+fjahr,
+friendly_arthrop,
+GeraldineG,
+gevs,
+gg34,
+ghost43,
+ghosthell,
+giaki3003,
+gloved,
+gmaxwell,
+graeme1,
+GreenmanPGI,
+gr-g,
+GVac,
+gwillen,
+gwj,
+gz12,
+gz77,
+h4shcash,
+harding,
+hebasto,
+hiro8,
+Hotmetal,
+hsjoberg,
+huesal,
+instagibbs,
+Ironhelix,
+IT4Crypto,
+ja,
+jaenu,
+JanB,
+jeremyrubin,
+jimmy53,
+jnewbery,
+jonatack,
+jonny100051,
+jtimon,
+kallewoof,
+kanon,
+kanzure,
+Kappa,
+keblek,
+ksedgwic,
+landeau,
+lucasmoten,
+luke-jr,
+maaku,
+Majes,
+maybehuman,
+mblackmblack,
+mcm-mike,
+Memesan,
+michaelfolkson,
+midnight,
+MikeMarzig,
+mips,
+mol,
+molz,
+moneyball,
+mrb07r0,
+MrHodl,
+murch,
+Murch,
+naribia,
+newNickName,
+nickler,
+nikitis,
+NoDeal,
+norisgOG,
+nothingmuch,
+occupier,
+OP_NOP,
+OtahMachi,
+p0x,
+pinheadmz,
+PinkElephant,
+pox,
+prayank,
+prepaid,
+proofofkeags,
+provoostenator,
+prusnak,
+qubenix,
+queip,
+r251d,
+rabidus,
+Raincloud,
+raj,
+RamiDz94,
+real_or_random,
+rgrant,
+riclas,
+roasbeef,
+robert_spigler,
+rocket_fuel,
+roconnor,
+rovdi,
+rubikputer,
+RusAlex,
+rusty,
+sanket1729,
+satosaurian,
+Satoshi,
+schmidty,
+sdaftuar,
+setpill,
+shesek,
+shinobiusmonk,
+snash779,
+solairis,
+somethinsomethin,
+stortz,
+sturles,
+sugarpuff,
+taPrOOteD,
+TechMiX,
+TheDiktator,
+thomasb06,
+tiagocs,
+tomados,
+tonysanak,
+TristanLamonica,
+UltrA1,
+V1Technology,
+vanity,
+viaj3ro,
+Victorsueca,
+virtu,
+walletscrutiny,
+wangchun,
+warren,
+waxwing,
+Whatisthis,
+whuha,
+willcl_ark,
+WilliamSantiago,
+windsok,
+wumpus,
+xxxxbtcking,
+yanmaani,
+yevaud,
+ygrtiugf,
+Yoghurt11411,
+zmnscpxj,
+and
+zndtoshi.
+</i>
+
+**Miner signaling**
+
+We also thank all of the miners since block FIXME that have signaled
```

Comment:
Taproot's speedy trial nStartTime was April 24th:

```
        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nStartTime = 1619222400; // April 24th, 2021
```

The first block for which the activation was STARTED state was therefore the next retarget block [681408](https://blockstream.info/block/000000000000000000020210114d7faf44223e1d620991e468af964514ff96e3?expand), mined on May 1st.